tests/testthat/test-climatic_summary.R

```diff
@@ -0,0 +1,25 @@
+library(RInstatClimatic)
+
+x <- tibble::as_tibble(data.frame(station_name = c("Agades","Birni N'Konni","Niamey_Aero","Zinder"), mean_rain = as.numeric(unlist(c(NA,NA,NA,NA))), st_dv_rain = as.numeric(unlist(c(NA,NA,NA,NA))), n_na_rain = as.integer(unlist(c(101, 88,  122, 112)))))
```

Comment:
It would be better do this as the equivalent `dplyr` code e.g.
```r
x <- daily_niger %>%
    group_by(station_name) %>%
    summarise(mean_rain = mean(rain),
              st_dv_rain = sd(rain),
              n_na_rain = naflex::na_n(rain)
    )
```
This way you don't have to enter values manually. And if the data frame changed in future, the tests don't need to be changed.